README.md

```diff
@@ -4,25 +4,66 @@ Command line tool to upload/manage project on the [duke-data-service](https://gi
 
 Runs on Python 2.7 or 3.5.
 
-###Install:
+# Install:
 ```
 pip install DukeDSClient
 ```
 
-### Additional Setup:
-To use DukeDSClient, you must supply an authentication token.
+# Additional Setup:
 This requires a [Duke NetID](https://oit.duke.edu/email-accounts/netid/).
-This token identifies you with the service, and can be generated by visiting the [Duke Data Service API Explorer](https://api.dataservice.duke.edu/apiexplorer).
-Click the Authenticate button.
-Click Duke Authentication Service.
-Enter your NetId credentials.
-Your authentication token will then be printed at the top of the screen.
+To use DukeDSClient, it requires an agent key and a user key to be added to a config file.
```

Comment:
This is awkwardly phrased. Perhaps "DukeDSClient requires a configuration file containing an **agent key** and a **user key**". Both can be obtained by visiting the Duke Data Sevice Portal website.